dev/release/utils-common.sh

```diff
@@ -38,20 +38,8 @@ header() {
 
 changelog() {
     # Strip trailing blank line
-    local -r changelog=$(printf '%s\n' "$(cz ch --dry-run --unreleased-version "ADBC Libraries ${RELEASE}" --start-rev apache-arrow-adbc-${PREVIOUS_RELEASE})")
-    # Split off header
-    local -r header=$(echo "${changelog}" | head -n 1)
-    local -r trailer=$(echo "${changelog}" | tail -n+2)
-    echo "${header}"
-    echo
-    echo "### Versions"
-    echo
-    echo "- C/C++/GLib/Go/Python/Ruby: ${VERSION_NATIVE}"
-    echo "- C#: ${VERSION_CSHARP}"
-    echo "- Java: ${VERSION_JAVA}"
-    echo "- R: ${VERSION_R}"
-    echo "- Rust: ${VERSION_RUST}"
-    echo "${trailer}"
+    local -r changelog=$(printf '%s\n' "$(env PYTHONPATH=${SOURCE_TOP_DIR}/dev python -m adbc_dev.changelog --name "ADBC Libraries ${RELEASE}" apache-arrow-adbc-${PREVIOUS_RELEASE} HEAD 2>/dev/null)")
+    echo "${changelog}"
```

Comment:
We may be able to simplify this:

```suggestion
    env PYTHONPATH=${SOURCE_TOP_DIR}/dev python -m adbc_dev.changelog --name "ADBC Libraries ${RELEASE}" apache-arrow-adbc-${PREVIOUS_RELEASE} HEAD 2>/dev/null
```

Reply:
I think that should work. I think the previous tool tended to add trailing blank lines in the output that I used `printf` to strip.